### PR TITLE
add back random hook

### LINKED ISSUE #1424: How to use a random number?
Hello, 
I want to generate a random number using the function defined in random.k.
I always get the following result : 
randomRandom(100)

i wonder what i am doing wrong ??

thank you very match for ansewring me.

### PR DESCRIPTION
**Pull request by dwightguth**
fixes #1424

@yilongli please review

### CODE REVIEW

File: k-distribution/include/builtins/builtins.k [hunk 1]
```diff
@@ -34,7 +34,7 @@ module BUILTIN-HOOKS
   imports FLOAT-HOOKS
   imports STRING-HOOKS
   imports K-EQUAL-HOOKS
-  //imports RANDOM-HOOKS // TODO: (andreis) fix the hook.
+  imports RANDOM-HOOKS // TODO: (andreis) fix the hook.
```
Comment (yilongli) [inline k-distribution/include/builtins/builtins.k]: Is this TODO item still valid?

File: java-backend/src/main/java/org/kframework/backend/java/builtins/BuiltinIntOperations.java [hunk 2]
```diff
@@ -2,8 +2,12 @@
 package org.kframework.backend.java.builtins;
 
 
+import jline.ANSIBuffer;
```
Comment (yilongli) [inline java-backend/src/main/java/org/kframework/backend/java/builtins/BuiltinIntOperations.java]: remove this?

**Comment by yilongli** [comment]
lgtm

File: k-distribution/include/builtins/builtins.k (hunk 1 above)
Comment (andreistefanescu) [inline k-distribution/include/builtins/builtins.k]: @radumereuta this was to avoid some parsing warning, right?

Comment (radumereuta) [inline k-distribution/include/builtins/builtins.k]: Yeah.

**Pull request merged**